cekit/templates/template.adoc.jinja

```diff
@@ -0,0 +1,150 @@
+{%- macro resource_location(resource) -%}
```

Comment:
this filename (template.adoc.jinja) should maybe more descriptive (module.adoc.jinja?)